tests/utils/date_parser_tests.py

```diff
@@ -88,10 +88,18 @@ def test_get_since_until(self):
         expected = datetime(2016, 6, 7), datetime(2016, 11, 7)
         self.assertEqual(result, expected)
 
+        result = get_since_until("Last 5 month")
+        expected = datetime(2016, 6, 7), datetime(2016, 11, 7)
+        self.assertEqual(result, expected)
+
```

Comment:
Could we make this UT test for the correct singular case "Last 1 month"? Having the "Last 5 month" test is probably a good additional test, but less important IMO.

Reply:
thanks for reminding me.  Because there is `Last 5 months` UT case now, so added a `Last 5 month`(This syntax is wrong )